quickevent/app/quickevent/plugins/Oris/src/xmlimporter.h

```diff
@@ -0,0 +1,65 @@
+#ifndef XMLIMPORTER_H
+#define XMLIMPORTER_H
+
+#include <QObject>
+#include <QXmlStreamReader>
+#include <QSet>
+#include <QMap>
+
+class XmlImporter : public QObject
+{
+	Q_OBJECT
+public:
+	enum class XmlCreators : int
+	{
+		QuickEvent = 0,
+		Oris,
+		Eventor,
+	};
+	struct SPerson
+	{
+		QString nameGiven;
+		QString nameFamily;
+
+		QString className;
+		QString classNameShort; // backup, if not defined className
+
+		int iofId = -1;
+		QString regCz;		// ORIS
+		QString noteOris;	// ORIS
+		int orisId = -1;	// ORIS
+
+		int siNumber = 0;
+
+		QString nationalityCode;	// Person
+		QString nationalityName;	// Person
+		QString clubCode;			// Organisation - Club
+		QString clubName;			// Organisation - Club
+		QString countryCode;		// Organisation - NationalFederation
+		QString countryName;		// Organisation - NationalFederation
+		int clubIdIof = -1;	// Eventor (Club or NationalFederation ID)
+		QSet<int> enterRaces;
+		// ...
+		int bib = 0;
+		int rank = 0;
+		int start_order_pref = 0; // 0-none /  1- early / 2- middle / 3- late
```

Comment:
enum please

Reply:
Currently unused variables (rank, start_order_pref) - will be removed.